data/category.go

```diff
@@ -31,10 +31,13 @@ func GetDataDirs(root, pattern string) []string {
 	return specs
 }
 
-func NewCategories(dirs []string) []spec.Spec {
+func NewCategories(dirs []string, filter string) []spec.Spec {
```

Comment:
I'm not sure about the Categories name either. Right now a Category contains a dataset and a collection of queries and results.

A `Suite` would be slightly more common language in testing, but I don't think that's quite exact either. 

Category is probably fine for now, but think about what might be a better name.